src/styles/_utils.scss

```diff
@@ -317,6 +317,18 @@ $sides: (top, right, bottom, left);
   place-self: flex-end !important;
 }
 
+/*  Position
```

Comment:
Can we document these new util classes?

https://deploy-preview-1117--kongponents.netlify.app/guide/styles/utilities.html

Reply:
This is obsolete